### PR TITLE
Added support for plots in sidecar if present.

### PR DESCRIPTION
**Pull request by doutriaux1**
Just to see what @chaosphere2112 did

### CODE REVIEW

**Comment by doutriaux1** [comment]
@downiec @scottwittenburg @aashish24 @sterlingbaldwin this is what @chaosphere2112 did to make the widgets magically appear in notebooks.

**Comment by doutriaux1** [comment]
@aashish24 @chaosphere2112 not sure on how to trigger this it does not seem to appear in the notebook when I plot.

File: vcs/displayplot.py [hunk 1]
```diff
@@ -108,6 +109,30 @@ def _repr_png_(self):
         f = open(tmp, "rb")
         st = f.read()
         f.close()
+        try:
+            import IPython.display
+            import cdat_notebook
```
Comment (doutriaux1) [inline vcs/displayplot.py]: @chaosphere2112 do you where this comes from? That's the failing line

Comment (doutriaux1) [reply vcs/displayplot.py]: Ok i found the repos I think

File: vcs/displayplot.py [hunk 2]
```diff
@@ -108,6 +109,30 @@ def _repr_png_(self):
...
+            if self.g_type == "boxfill":
+                box = vcs.getboxfill(self.g_name)
+                b_dict = vcs.utils.dumpToDict(box)[0]
+                if self._widget is not None:
+                    self._widget.close()
+                self._widget = cdat_notebook.GMWidget(value=b_dict)
```
Comment (doutriaux1) [inline vcs/displayplot.py]: followed the instructions to enable cdat_notebooks-widget extension but the above line still return None for self._widget

Comment (doutriaux1) [reply vcs/displayplot.py]: @aashish24 @downiec @scottwittenburg if you have some time to look into this it would be a good stsarting point for v2.0

Comment (doutriaux1) [reply vcs/displayplot.py]: repo is at: https://github.com/CDAT/cdat-notebook-widgets

**Comment by doutriaux1** [comment]
@muryanto1 this branch fixes `master` and does not timeout on coverage build. 
@sterlingbaldwin @downiec @scottwittenburg please review and let's merge this in master so that vcdat 2.0 can use nightlies and put plots in sidecar.

**Comment by doutriaux1** [comment]
Also in this, if you write `no_coverage` in your commit it will skip the coverage which makes build a bit faster and is usefull when debugging

**Comment by doutriaux1** [comment]
Note to test sidecar on your end
```
conda install -c conda-forge pip nodejs
pip install sidecar
jupyter labextension install @jupyter-widgets/jupyterlab-manager
jupyter labextension install @jupyter-widgets/jupyterlab-sidecar
```

**Comment by doutriaux1** [comment]
@sterlingbaldwin @scottwittenburg @downiec ping!

File: .circleci/config.yml [hunk 3]
```diff
@@ -91,7 +101,14 @@ aliases:
     command: |
        export PATH=$WORKDIR/miniconda/bin:$PATH
        source activate $PY_VER
-       coveralls
+       # possibly turn off coverage
+       first=$(git cherry master -v | head -n1)
+       current=$(git log -n1 --pretty=oneline)
+       if [[ $first == $current || $current != *"no_coverage"*  ]]; then
```
Comment (scottwittenburg) [inline .circleci/config.yml]: @doutriaux1 Can you provide a quick high-level description of what you're trying to capture here?  I don't think I understand the git workflow you have in mind for turning off coverage with commits.

Comment (doutriaux1) [reply .circleci/config.yml]: just if you have "no_coverage" in commit and it's not the first commit on the PR then skip coverage.

Comment (doutriaux1) [reply .circleci/config.yml]: because coverage is slow

**Pull request merged**